permission-portal-cloud-functions/functions/__tests__/auth.test.ts

```diff
@@ -355,7 +355,7 @@ test('User can be toggled between enabled and disabled', () => {
     })
     .then(() => {
       // Delay to allow userOnUpdate time to run
-      return delay(DELAY).then(() => {
+      return delay(DELAY * 2).then(() => {
```

Comment:
seems odd. Intentional?

Reply:
Yes, these tests are relying on Firebase free-tier infrastructure, which can take some time to "boot up" and propagate these changes. Not ideal but necessary to test this functionality.